operations/configuration_management/iSCSI_SBPS_Node_Personalization.md

```diff
@@ -0,0 +1,357 @@
+# iSCSI SBPS worker node(s) personalization
+
+As part of the iSCSI SBPS solution, we need to setup/ configure worker nodes as iSCSI targets (servers) with necessary provisioning, configuration and enablement of required components.
+In order to achieve this, CFS based Ansible plays (`csm-config`) are created and deployed in order to
+
+* provision `iSCSI targets`/`LIO services` and present the `LIO network` service via the HSN and NMN IP networks (TCP port 3260 by default), via iSCSI Portals.
+  
+* enable SBPS Marshal Agent by installing the agent rpm and starting the respective `systemd` service (`sbps-marshal`).
+  
+* enable `spire` for authentication used for `IMS/ S3` access.
+  
+* create `DNS SRV and A records` in order to discover iSCSI targets from iSCSI initiators/clients (passed as boot parameter).
+  
+* mount S3 images (`boot-images` bucket) with new, dedicated S3 user (`ISCSI-SBPS`) read only bucket policy via `s3fs`.
+  
```

Comment:
The blank lines have trailing whitespace, please remove them.

Reply:
Fixed.